BasicGates/WorkBook_BasicGates1-4_VM3.ipynb

```diff
@@ -0,0 +1,983 @@
+    "In the example given in the Kata, notice how the amplitudes of the two basis states are normalised."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "promotional-oxford",
+   "metadata": {},
+   "source": [
+    "**Note:** that although the Kata gives you the answer here, it is well worth recalling that the Pauli-X gate is a fundamental one which is used in combination with many other gates to achieve a wide variety of different changes of qubit(s) state(s). Recall that the other two Pauli gates are the Pauli-Y and the Pauli-Z gates. All these gates are self -adjoint (i.e.applying them twice returns the qubit to its original state)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "sustained-claim",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%kata T101_StateFlip \n",
+    "\n",
+    "operation StateFlip (q : Qubit) : Unit is Adj+Ctl {\n",
+    "    // The Pauli X gate will change the |0⟩ state to the |1⟩ state and vice versa.\n",
+    "    // Type X(q);\n",
+    "    // Then run the cell using Ctrl+Enter (⌘+Enter on macOS).\n",
+    "    X(q);\n",
+    "    // ...\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "disturbed-wilson",
+   "metadata": {},
+   "source": [
+    "[Return to Task 1 of the Basic Gates Quantum Kata Notebook  (opens in new tab)](./BasicGates.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "satisfied-volunteer",
+   "metadata": {},
+   "source": [
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "loved-korean",
+   "metadata": {},
+   "source": [
+    "## Task 1.2. Basis change: |0⟩ to |+⟩ and |1⟩ to |-⟩ (and vice versa)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "destroyed-scoop",
+   "metadata": {},
+   "source": [
+    "**Input:** A qubit in state |ψ⟩ = α |0⟩ + β |1⟩.\n",
+    "\n",
+    "**Goal:** Change the state of the qubit as follows:\n",
+    "If the qubit is in state |0⟩, change its state to |+⟩ = (|0⟩ + |1⟩) / sqrt(2).\n",
+    "If the qubit is in state |1⟩, change its state to |-⟩ = (|0⟩ - |1⟩) / sqrt(2).\n",
+    "If the qubit is in superposition, change its state according to the effect on basis vectors, as specified above."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "virtual-recall",
+   "metadata": {},
+   "source": [
+    "## Solution"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "twenty-proportion",
+   "metadata": {},
+   "source": [
+    "This is fundamentally book work, and a well-known property of the extremely important Hadamard (H) gate.  \n",
```

Comment:
This paragraph says the same thing in several different ways, but doesn't give a reminder of how the H gate looks like, like the next task. 

Generally, it would be nice to come up with a unified way to explain the solutions to the tasks in this kata that require only one gate. For example, 1) mention that a certain gate can be recognized, 2) give the matrix representation of the gate, 3) work out an example of applying this gate to some state, or something like that

(And then we get variety in the extension tasks because they each ask for things of different types)